lib/PINNBench/Burgers/requirements.txt

```diff
@@ -1,6 +1,5 @@
-deephyper==0.4.2
+deephyper==0.5.0
```

Comment:
I suggest replacing with `deephyper>=0.5.0` to be less restrictive. It is less safe that's true... so maybe we should have some `install(..., strict=True/False)` to be able to install the "strict" requirements (tested) and something a bit softer.